build/common/installer/scripts/tomlparser-agent-config.rb

```diff
@@ -104,6 +104,11 @@
 @waittime_port_13000 = 45 # default waittime for AMACA data port
 @waittime_port_12563 = 45 # default waittime for AMACA config port
 
+@netoworkFlowLogsThrottleRate = 1000
```

Comment:
typo. This should be network instead of netowork.

Reply:
changed